### ISSUE TITLE
Need Documentation/article

### ISSUE THREAD

**Issue by JiteshPshah** [akash-suresh/TheChainRanker#34]
Hi, 
Have you make  documentation/research paper on how you implemented whole system ?

**Comment by akash-suresh** [comment]
Yeah. We have a detailed paper on the implementation. Please drop us a mail, and we'd be happy to share it with you. 
Drop a mail to akashzsuresh@gmail.com.

**Comment by angeluau** [comment]
thanks for your share ,my email is 2215346277@qq.com

**Comment by LordSnow1992** [comment]
Hi can you share the documentation with me as well my e-mail is jonthesnow666@gmail.com
